feature-libs/product-configurator/rulebased/components/attribute/types/multi-selection-bundle/configurator-attribute-multi-selection-bundle.component.html

```diff
@@ -10,7 +10,8 @@
     *ngFor="let value of attribute.values"
     [product]="value"
     [multiSelect]="true"
-    [disabledAction]="disableDeselectAction$ | async"
+    [disabledAction]="false"
```

Comment:
are we explicitly setting this to `false`? or is it a mistake

Reply:
it is no mistake, we dont need more this setting, i will remove all code related to it